exercises/security/README.md

```diff
@@ -423,3 +414,116 @@ Practically this means that a single IP can server multiple web services/pages,
+</b></details>
+
+<details>
+<summary> Suggest and describe a specific tool that could enhance threat prevention from a developer perspective. 
+</summary><br><b>
+
+[Aquasec](https://www.aquasec.com/aqua-cloud-native-security-platform/): 
+Aquasec is a tool which scans a project for potential security threats. This involves for instance vulnerability scanning, cloud security, Kubernetes security and serverless security.
+</b></details>
+
+#### Package management & Security 
+
+<details>
+<summary> What is a package manager?
+</summary><br><b>
+
+[Baudry et al.](https://arxiv.org/pdf/2001.07808.pdf): A tool that allows you to easily download, add and thus reuse programming libraries in your project. E.g. npm or yarn.
+</b></details>
+
+<details>
+<summary> What is a build tool?
+</summary><br><b>
+
+[Baudry et al.](https://arxiv.org/pdf/2001.07808.pdf): A tool that fetches the packages (dependencies) that are required to compile, test and deploy your application. 
+</b></details>
+
+<details>
+<summary> Describe bloated dependencies.
+</summary><br><b>
+
+[Baudry et al.](https://arxiv.org/pdf/2001.07808.pdf): 
+An application usually has different dependencies. Typically, not all of them are required for building and running the application. Bloated dependencies is the concept of including the unnecessary dependencies for building and running your application. 
+</b></details>
+
+<details>
+<summary> Explain a few cons with bloated dependencies.
+</summary><br><b>
+
+[Baudry et al.](https://arxiv.org/pdf/2001.07808.pdf): 
+
+  * Challenging to manage.
+  * Decreases performance of the application.
+  * Risk for malicious code that a threathening actor can take advantage of. 
+</b></details>
+
+<details>
+<summary> Briefly explain how DepClean for Maven projects work.
+</summary><br><b>
+
+[Baudry et al.](https://arxiv.org/pdf/2001.07808.pdf): You input your project consisting of bytecode, a project object model (POM) and the Maven central repository into DepClean. DepClean will identify the dependency tree and perform bytecode analysis of API member calls in order to determine what dependencies are actually used in the project. The used dependencies will be intact, whereas the idle dependencies are removed from the dependency tree. DepClean will ultimately produce a dependency usage report and a debloated POM file. 
+</b></details>
+
+<details>
+<summary> Provide another solution to manage dependencies. 
+</summary><br><b>
+
+[Npm.js documentation](https://docs.npmjs.com/cli/v8/commands/npm-prune): Use clean-up commands that are usually provided by the package manager authors. For instance, npm prune will remove any extraneous package. Another command is npm audit which will scan your repository and report any vulnerable dependencies found.
+</b></details>
+
+<details>
+<summary> What is a threatening actor and how can this actor take advantage of open source or third party vendor's packages/libraries? 
```

Comment:
I would add a question of what is an actor in first place

Reply:
"In first place" - do you mean in the beginning of the 'software supply chain' section or right before this question? 

We think that a question of what a threatening actor is should be enough for the reader to understand the question and answer, given the context (security) and the question's nature. Also, we believe that a question about what an actor is (generally speaking?) might not fit the context, but those are just our thoughts. In case you want a question about what an actor is, we'll of course add it (if so, please let us know in what context).